automatic/sysinternals/tools/chocolateyBeforeModify.ps1

```diff
@@ -0,0 +1,2 @@
+﻿$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+Remove-Process -PathFilter ([regex]::escape('C:\ProgramData\chocolatey\lib\sysinternals\tools') + ".*") | Out-Null
```

Comment:
Where have you found the usage of `Remove-Process` from?

I can not find any reference to this cmdlet, additionally please add a call to `Write-Host` that mentions processing being Stopped/Removed.

Reply:
It's part of `chocolatey-core.extension`
https://github.com/chocolatey-community/chocolatey-extensions/blob/master/src/chocolatey-core.extension/extensions/Remove-Process.ps1

I've added a Write-Host call.

Reply:
Ah, I had completely missed that the extension had it.